lib/engine/round/operating.rb

```diff
@@ -237,6 +244,14 @@ def next_step!
         end
       end
 
+      def skip_home_token
+        if @home_token_timing == :operate
```

Comment:
return !place_home_token(@current_entity) if @home_token_timeing == operati

Reply:
I've changed it to a secondary check